### PR TITLE
Milestone5 full website

### PR DESCRIPTION
**Pull request by miftah1991**
This is 5th milestone of the math magicians project, in this milestone below features has been added

- Create 3 pages: Home, CalculatorPage, and Quote.
- Create Header.jsx components that contain the Navigation.
- Install a third-party package called react-router-dom to render the pages.
- Create a style  folder that contains all React basic styling.

### CODE REVIEW

File: src/pages/CalculatorPage.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import '../style/CalculatorPage.css';
+
+const CalculatorPage = () => (
+  <section className=" container calculator-section">
+    <h2> Let&#39;s do some math ! </h2>
+    <div className="calculator">
+      <Calculator />
+    </div>
+  </section>
+);
```
Comment (DeliceLydia) [inline src/pages/CalculatorPage.jsx:12]: - [ ] I suggest that you add a color that makes it easier to see numbers on your calculator this is how it looks like now:
cal

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @miftah1991, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Here are major highlights for your project:
- No linter errors
- Followed git workflow
- Implemented all pages required


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @miftah1991, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by miftah1991** [comment]
@Amrhub 

Thank you so much

**Pull request merged**